LICENSE

```diff
@@ -1,181 +1,193 @@
+---
+weight: 504
+toc: true
+title: License
+menu:
+  docs:
+    parent: community
+lead: ""
+images: []
+draft: false
+---
+
```

Comment:
I'm not sure if we're allowed to make changes to the LICENSE file 🤔

Reply:
Without this, we won't be able to include it. Should I remove it?